pom.xml

```diff
@@ -14,8 +14,8 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
     <!-- connector SDK version -->
-    <version.connector-core>0.23.2</version.connector-core>
-    <version.connector-runtime>0.23.2</version.connector-runtime>
+    <version.connector-core>8.3.0</version.connector-core>
+    <version.connector-runtime>8.3.0</version.connector-runtime>
```

Comment:
This version is obsolete - while we're here, could you please remove it and use the same version everywhere?